### ISSUE TITLE
Display Window Geometry Configuration

### ISSUE THREAD

**Issue by Defreezer** [stenzek/duckstation#2578]
**Is your feature request related to a problem? Please describe.**
Feature request.

I have noticed that Duckstation's settings.ini produces strings of values under the UI section named "MainWindowGeometry" and "DisplayWindowGeometry" whenever the size of the emulator window is modified. Interestingly, when paired up with external software like Borderless Gaming, which expands its Qt window borders beyond the edges of the display itself without the need to go fullscreen, this geometry modification is likewise saved in the .ini and can be reset by clearing the Geometry values in the file, thus resetting them to their default size.

**Describe the solution you'd like**
Would it be in your interest to include window width/height resolution options in the Display section? Perhaps allow it to accept direct input of custom values in the settings.ini as to neatly accomodate specific aspect ratios/internal scaling (e.g.: "DisplayWindowGeometry = 606x478" for a 4:3 aspect ratio) or arbitrarily expand the window borders beyond the display for a borderless window option (e.g.: "DisplayWindowGeometry = 1928x1110")? Given that, presently, Duckstation seems to expand the internal rendering as much as possible within the constraints of its display window, modifying its resolution more accurately via the settings than resizing window borders with the mouse could be an interesting addition.

**Comment by BailShamber** [comment]
There's already an option for borderless fullscreen in duckstation's display settings. Is there a reason why "borderless gaming" might be preferable over it for you?

**Comment by Defreezer** [comment]
I've only used Borderless Gaming for a few tests, and haven't since, though I noticed that the Geometry value was maintained after I removed the program. Applying BG to the main window omits the borders and places it over the taskbar, making full use of the display without the need to toggle fullscreen, all while keeping the toolbar visible at all times. Going from the faux-fullscreen of the separate rendering window to the rom list to change settings felt more seamless in comparison to the fullscreen toggling function. It yielded some interesting results, I wonder if such a thing could be achieved without the need for third-party programs such as BG?
![test](https://user-images.githubusercontent.com/79490923/132073496-e6a52ea8-cccb-4980-8e12-810a616e57d4.jpg)
